modules/lib/lib-node/src/main/resources/lib/xp/examples/node/query-dsl-sort.js

```diff
@@ -0,0 +1,56 @@
+var nodeLib = require('/lib/xp/node');
+var assert = require('/lib/xp/testing');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=8706041)
```suggestion
var assert = require("/lib/xp/testing");
```